frame/bounties/src/tests.rs

```diff
@@ -109,8 +107,8 @@ parameter_types! {
 	pub const Burn: Permill = Permill::from_percent(50);
 	pub const DataDepositPerByte: u64 = 1;
```

Comment:
can be removed ? also SpendPeriod and ProposalBondMinimum above ?

Reply:
Alright